### ISSUE TITLE
Podporované formáty souborů

### ISSUE THREAD

**Issue by kalich5** [skaut/skaut-google-drive-documents#11]
Můžeme dát dohromady seznam toho, co všechno ten plugin dokáže zobrazit. Aby se to dalo dát do nějakého popisu. 
Děkuji

**Comment by kalich5** [comment]
Grafik co bude dělat logo a obrázky pro WordPress.org by chtěl vědět ty formáty co to podporuje. Aby to mohl zapracovat do grafiky. Chce to dělat zítra, tak bych uvítal odpověď co nejdříve. :-)

**Comment by marekdedic** [comment]
Ahoj,
mně by se tam líbily:
- Formuláře
- Dokumenty
- Tabulky
- Prezentace
- Mapy

Všechno myslím takové ty "nativní" pro Drive. Priorita je podle mě v tom pořadí, v jakém jsem to napsal (Formuláře nejdůležitější).

Nevím, zda jde přes to API nějak zobrazit i klasické Microsoft Office/OpenDocument, pokud ano, tyk bych to bral jen jako bonusovou feature, ne jako prioritu.

**Comment by xkosorin** [comment]
Podľa všetkého by to malo podporovať všetky dokumenty ktoré podporuje samotný gdrive. Tu by mal byť kompletný zoznam: [File formats](https://en.wikipedia.org/wiki/Google_Drive#File_viewing). Typy ktoré som zatiaľ testoval na zobrazenie sú: Google Docs (dokument, formular, prezentacia, tabulka), PDF, DOCX, XLSX

**Comment by kalich5** [comment]
Super, děkuji za info

**Comment by marekdedic** [comment]
Jo, PDF určitě taky. S těma Wordovýma bych to zatím nehrotil, pokud by to byl problém...

**Comment by kalich5** [comment]
Microsoft formáty jsou důležité, na diskách jich je spousta. 

Microsoft formáty, PDF, Google formáty... a další formáty :-)

**Comment by xkosorin** [comment]
Nefunguje zobrazenie ZIP súborov. Príde mi to trochu ako chyba zo strany Google ale stále sa snažím zistiť či je to chyba alebo fearute
